sql/src/stream.cc

```diff
@@ -714,12 +714,12 @@ void stream::_process_downtime(
                "      actual_start_time=COALESCE(actual_start_time, :actual_start_time),"
                "      author=:author, cancelled=:cancelled, comment_data=:comment_data,"
                "      deletion_time=:deletion_time, duration=:duration, end_time=:end_time,"
-               "      fixed=:fixed, instance_id=:instance_id, internal_id=:internal_id,"
-               "      start_time=:start_time, started=:started, triggered_by=:triggered_by,"
-               "      type=:type"
+               "      fixed=:fixed, start_time=:start_time,"
+               "      started=:started, triggered_by=:triggered_by,"
+               "      type=:type, host_id=:host_id, service_id=:service_id"
```

Comment:
Could you reorder fields in the update query so that they are sorted in alphabetical order ? This helps checking queries in case of error :-)